src/time/time.controller.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  Controller,
+  Post,
+  Get,
+  HttpCode,
+  HttpStatus,
+  Body,
+  Param,
+  Put,
+  Delete,
+} from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { TimeService } from './time.service';
+import { Public } from 'src/auth/public-strategy';
+import { CreateTimeDto } from './dto/create-time.dto';
+import { UpdateTimeDto } from './dto/update-time.dto';
+
+@ApiTags('Time')
+@Controller('time')
+export class TimeController {
+  constructor(private readonly timeService: TimeService) {}
+
+  @Public()
+  @Post(':userId/create')
+  @HttpCode(HttpStatus.OK)
```

Comment:
(suggestion; non-blocking): It can be standard practice to use the 201 HTTP status code for `POST` requests - https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201